core/egov-localization/src/main/java/org/egov/persistence/repository/builder/MessageRepositoryQueryBuilder.java

```diff
@@ -2,17 +2,19 @@
 
 import java.util.List;
 
-import org.egov.web.contract.CreateMessagesRequest;
+import org.egov.domain.model.Message;
 import org.springframework.stereotype.Component;
 
 @Component
 public class MessageRepositoryQueryBuilder {
-	public String getDeleteQuery(CreateMessagesRequest createMessagesRequest){
-		List<org.egov.web.contract.Message> messageList = createMessagesRequest.getMessages();
-		String deleteQuery = "DELETE FROM message WHERE locale = ? AND tenantid = ? AND code IN  ";
+	public String getDeleteQuery(List<Message> messageList){
+		String deleteQuery = "DELETE FROM message WHERE locale = ? AND tenantid = ? AND code IN ";
```

Comment:
@darshannagesh Combination of locale, tenant, code and module uniquely points to a message. You would also need to have module column part of the where condition.